api_payments.yaml

```diff
@@ -0,0 +1,286 @@
+          schema:
+            $ref: "#/definitions/PaymentsCheckRequest"
+          x-examples:
+            application/json:
+              codiceIdRPT:
+                CF: "TMMEXQ60A10Y526X"
+                AuxDigit: 1
+                CodIUV: 123456789012345
+                CodStazPA: 56
+      responses:
+        '200':
+          description: Payment information retrieved
+          schema:
+            "$ref": "#/definitions/PaymentsCheckResponse"
+          examples:
+            application/json:
+              importoSingoloVersamento: 200,
+              codiceContestoPagamento: "ABC123"
+        "400":
+          description: Input provided or pagoPa response are invalid
+        "500":
+          description: PagoPa services are not available or request is rejected
+  "/payment/activation":
+    x-swagger-router-controller: PaymentController
+    get:
+      operationId: activatePaymentToPagoPa
+      summary: Activate Payment
+      description: Require a lock (activation) for a payment (qrcode)
+      parameters:
+        - in: body
+          name: body
+          schema:
+            $ref: "#/definitions/PaymentsActivationRequest"
+          x-examples:
+            application/json:
+              codiceIdRPT:
+                CF: "TMMEXQ60A10Y526X"
+                AuxDigit: 1
+                CodIUV: 123456789012345
+                CodStazPA: 56
+              importoSingoloVersamento: 200
+              codiceContestoPagamento: "ABC123"
+      responses:
+        '200':
+          description: Payment Activation process started
+          schema:
+            "$ref": "#/definitions/PaymentsActivationResponse"
+          examples:
+            application/json:
+              importoSingoloVersamento: 200
+        "400":
+          description: Input provided or pagoPa response are invalid
+        "500":
+          description: PagoPa services are not available or request is rejected
+  "/payment/activation/check/{codiceContestoPagamento}":
+    x-swagger-router-controller: PaymentController
+    get:
+      operationId: checkPaymentActivationStatusFromDB
+      summary: Check Activation status
+      description: Check the activation status to retrieve the paymentId
+      parameters:
+        - name: codiceContestoPagamento
+          in: path
+          type: string
+          required: true
+          description: The session token for PagoPa
+      responses:
+        '200':
+          description: Payment information retrieved
+          schema:
+            $ref: "#/definitions/PaymentsActivationStatusCheckResponse"
+          examples:
+            application/json:
+              idPagamento: "XYZ123234234"
+        "400":
+          description: Input provided is invalid
+        "500":
+          description: Activation status is not available
+definitions:
+  FiscalCode:
+    $ref: "https://raw.githubusercontent.com/teamdigitale/digital-citizenship-functions/v0.40.0/api/definitions.yaml#/FiscalCode"
```

Comment:
last is https://raw.githubusercontent.com/teamdigitale/digital-citizenship-functions/v0.43.0/api/definitions.yaml